lte/gateway/python/magma/pipelined/rule_mappers.py

```diff
@@ -101,13 +101,14 @@ def update_all_ue_versions(self, imsi: str, ip_addr: IPAddress):
         """
         encoded_imsi = encode_imsi(imsi)
         if ip_addr is None or ip_addr.address is None:
-            ip_addr_str = ""
+            ip_addr_str = None
```

Comment:
Can you keep it "", when saving I used the empty str (helps with cwf compat)
https://github.com/magma/magma/blob/3f76e5fae724102b30debc5e52e72ca09f40d11d/lte/gateway/python/magma/pipelined/rule_mappers.py#L122

Reply:
sure